plugins/command-not-found/command-not-found.plugin.zsh

```diff
@@ -23,3 +23,13 @@ if [ -f /usr/libexec/pk-command-not-found ]; then
         return $retval
     }
 fi
+
+# OSX command-not-found support
+# https://github.com/Homebrew/homebrew-command-not-found
+if [ -e /usr/local/bin/brew ]; then
+  if brew tap | command grep -q homebrew/command-not-found; then
```

Comment:
This is unnecessary as it's already covered by `brew command command-not-found-init`